strawberryfields/ops.py

```diff
@@ -1286,6 +1340,40 @@ def _apply(self, reg, backend, **kwargs):
         backend.thermal_loss(p[0], p[1], *reg)
 
 
+class MbSgate(Channel):
```

Comment:
Just a thought: how about `MBSgate`? just because `B` also stands for a word.

Reply:
Also, this gate needs to be added to `channels` (doesn't appear in the docs atm)

Reply:
It almost sounds like `M-BSgate`. Maybe just `MSgate`?

Reply:
Hmm, could that be confused with the [Mølmer–Sørensen gate](https://en.wikipedia.org/wiki/M%C3%B8lmer%E2%80%93S%C3%B8rensen_gate) (not CV though)? Also, if the original `MbSgate` seems like the best candidate, I'm happy to second to that 😉

Reply:
Yeah that one is only for qubits, so no chance for confusion.

Reply:
I'll change to MSgate